operations/flags.go

```diff
@@ -144,6 +153,41 @@ func addMigrationRuntimeFlags(flags ...cli.Flag) []cli.Flag {
 
 }
 
+func addDbSettingsFlags(flags ...cli.Flag) []cli.Flag {
+	return append(flags,
+		cli.StringFlag{
+			Name:  dbUrlFlagName,
+			Usage: "Database URL(s). For a replica set, list all members separated by a comma.",
+			Value: evergreen.DefaultDatabaseUrl,
+		},
+		cli.BoolFlag{
+			Name:  dbSslFlagName,
+			Usage: "True to use SSL in the DB connection",
+		},
+		cli.StringFlag{
+			Name:  dbNameFlagName,
+			Usage: "Database name",
+			Value: evergreen.DefaultDatabaseName,
+		},
+		cli.IntFlag{
+			Name:  dbWriteNumFlagName,
+			Usage: "Number of mongod instances that need to acknowledge a write",
+		},
+		cli.StringFlag{
+			Name:  dbWmodeFlagName,
+			Usage: "Write mode. Only valid values are blank or 'majority'",
+		},
+		cli.BoolFlag{
+			Name:  dbFsyncFlagName,
+			Usage: "True if pending writes should flush to disk",
+		},
+		cli.BoolFlag{
+			Name:  dbJournalAckFlagName,
+			Usage: "True if the journal must acknowledge writes",
+		},
+	)
```

Comment:
Let's not give people the choice to turn off fsync/journal (fsync doesn't make sense after mmap) 

also the URL should be setable via an environment variable defined here.

Reply:
also we should make sure that these options, if set in the mongodb uri have clear semantics when they're specified by flags.

Reply:
I removed fsync/journal and also let the MONGO_URL env variable override the default URL

It looks like mgo doesn't support setting the write concern with the URL so I left the current write concern flags as is

Reply:
https://docs.mongodb.com/manual/reference/connection-string/#write-concern-options

Reply:
Let's keep it like this for now for backwards compatibility, and I'll open a ticket to switch this to use just the connection string